addons/survey/wizard/survey_invite.py

```diff
@@ -165,6 +165,11 @@ def _compute_body(self):
                 invite = invite.with_context(lang=langs.pop())
             super(SurveyInvite, invite)._compute_body()
 
+    @api.depends('template_id.attachment_ids')
+    def _compute_attachment_ids(self):
+        for invite in self:
+            invite.attachment_ids = invite.template_id.attachment_ids
```

Comment:
Could reset if we remove the template to avoid keeping part of the template (normally subject and body are reset when removing the template to start from scratch again)